sdk/azcore/poller.go

```diff
@@ -0,0 +1,453 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcore
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	headerOperationLocation = "Operation-Location"
+	headerLocation          = "Location"
```

Comment:
Although I see these are unexported, just asking if these should be organized along with the other headers in the headers.go file? Totally fine to leave it here though since they're just used for the pollers at the moment.

Reply:
Mostly I think of headers.go as the location for all of the ones we export.  However, perhaps we should place the unexported ones here too, to reduce the potential to duplicate them.

Reply:
I like that idea...simply because I could see duplicating headers if not placed in that file. Of course these are unexported so up to you, but I do think it's better